config/initializers/active_hash.rb

```diff
@@ -1,10 +1,8 @@
 Rails.application.config.to_prepare do
-  unless defined?(DATA_SOURCES)
-    DATA_SOURCES = {
-      "connectors" => GasAssets::Connector,
-      "pipes"      => GasAssets::Pipe,
-    }.freeze
-  end
+  DATA_SOURCES ||= {
```

Comment:
@antw I had some issues with the `unless defined`. Mainly because my `GasAssets` would be empty from time to time on development but with this code it isn't.